backend/src/apis/mod.rs

```diff
@@ -23,6 +23,7 @@ where
     let entry: Entry<N_CURRENCIES> = Entry::new(username, balances.try_into().unwrap()).unwrap();
 
     // Convert Fp to U256
-    let hash_str = format!("{:?}", entry.compute_leaf().hash);
-    U256::from_str_radix(&hash_str, 16).unwrap()
+    // TODO: Instead of hash of userdata, What should we out to this function?
```

Comment:
Could you please clarify this TODO? I think we can just remove the functionality related to the MST.

Reply:
I think it's left over in previous commit.